.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ jobs:
       xcode: "11.0.0"
     environment:
       - BAZEL_STARTUP_ARGS: "--output_base /Users/distiller/.cache/bazel"
-      - BAZEL_BUILD_ARGS: "--local_ram_resources=12288 --local_cpu_resources=8 --verbose_failures --test_env=ENVOY_IP_TEST_VERSIONS=v4only --test_output=all -c opt "
+      - BAZEL_BUILD_ARGS: "--local_ram_resources=12288 --local_cpu_resources=8 --verbose_failures --test_env=ENVOY_IP_TEST_VERSIONS=v4only --test_output=all -c fastbuild"
```

Comment:
@PiotrSikora Can you help fixing the mac build? I want to align mac with the prow script so they both produce fastbuild output.

Reply:
I'm so confused. Where does the binary live on mac in output_path?

Reply:
macOS builds use `darwin-{dbg,fastbuild,opt}`, so you can't align them with Prow.

We can't we simply use `bazel-bin`?

Reply:
darwin-opt doesn't work :( Prow doesn't have symlinks for bazel-bin, and using `bazel info bazel-bin` is incorrect since it always points to fastbuild.

Reply:
I thought links were fixed already, were they not?

Reply:
I thought so, too, but they are not. Can you tell me what I need to set for mac bazel path? I don't have a mac.

Reply:
Ok, I finally got the paths right.